contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -493,6 +589,8 @@ contract NodeOperatorsRegistry is AragonApp, Versioned {
         }
         _saveOperatorStuckPenaltyStats(_nodeOperatorId, stuckPenaltyStats);
         emit StuckValidatorsCountChanged(_nodeOperatorId, _stuckValidatorsCount);
+
```

Comment:
For `_updateStuckValidatorsCount()`:
- line 584: if stuckValidatorsCount is not cumulative `_requireValidRange()` check is incorrect. Must be less than active validators count
- maybe don't allow a decrease of stuck validators counter and increase refunded instead?
- No event on stuck penalty end timestamp value change

Reply:
1) both counters are not cumulative, i.e., they can decrease.
_requireValidRange - will fix to '<= DEPOSITED-EXITED`
2) maybe) 🤷🏻‍♂️
3) fixed